contracts/cw-ics721-bridge/src/contract.rs

```diff
@@ -144,28 +145,31 @@ fn receive_nft(
 ) -> Result<Response, ContractError> {
     let sender = deps.api.addr_validate(&sender)?;
     let msg: IbcOutgoingMsg = from_binary(&msg)?;
-
-    let class_id = if NFT_CONTRACT_TO_CLASS_ID.has(deps.storage, info.sender.clone()) {
-        NFT_CONTRACT_TO_CLASS_ID.load(deps.storage, info.sender.clone())?
-    } else {
-        let class_id = ClassId::new(info.sender.to_string());
-        // If we do not yet have a class ID for this contract, it is a
-        // local NFT and its class ID is its conract address.
-        NFT_CONTRACT_TO_CLASS_ID.save(deps.storage, info.sender.clone(), &class_id)?;
-        CLASS_ID_TO_NFT_CONTRACT.save(deps.storage, class_id.clone(), &info.sender)?;
-        // We set class level metadata to None for local NFTs.
-        //
-        // Merging and usage of this PR may change that:
-        // <https://github.com/CosmWasm/cw-nfts/pull/75>
-        CLASS_ID_TO_CLASS_URI.save(deps.storage, class_id.clone(), &None)?;
-        // TODO: set metadata
-        class_id
+    let class = NFT_CONTRACT_TO_CLASS.may_load(deps.storage, info.sender.clone())?;
+
+    let class = match class {
```

Comment:
```suggestion
    let class = match NFT_CONTRACT_TO_CLASS.may_load(deps.storage, info.sender.clone())? {
```

`match class` is probably less descriptive than just matching on what were doing. no need for indirection here imo